RELEASE-NOTES.txt

```diff
@@ -1,5 +1,9 @@
 18.9
 -----
+* [**] Block editor: Fix scroll update when typing in RichText component. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4299]
```

Comment:
I think we can change this to be clearer to non-devs:
> Fix issue where editor doesn't auto-scroll so you can see what is being typed

Reply:
Wdyt @twstokes?

Reply:
Sounds good @guarani - good call. Updated.